pkg/apis/serving/v1alpha1/knativeserving_lifecycle.go

```diff
@@ -48,11 +48,11 @@ func (is *KnativeServingStatus) InitializeConditions() {
 	conditions.Manage(is).InitializeConditions()
 }
 
-func (is *KnativeServingStatus) MarkInstallFailed(msg string) {
+func (is *KnativeServingStatus) MarkInstallFailed(format, msg string) {
 	conditions.Manage(is).MarkFalse(
 		InstallSucceeded,
 		"Error",
-		"Install failed with message: %s", msg)
+		format+": %s", msg)
```

Comment:
This feels like overkill. Why not construct the entire message clientside? You're effectively doing that anyway, except of the `:` that's added automatically.